<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="voice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Voice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewWindow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NewWindow</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fn_lst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="termButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="activeWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>wd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_for_newtab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>button_for_newtab</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
